### PR TITLE
Links s3

### PR DESCRIPTION
**Pull request by tylershuster**
dupe #3367

### CODE REVIEW

File: pkg/interface/src/views/apps/links/app.js [hunk 1]
```diff
@@ -152,6 +152,22 @@ export class LinksApp extends Component {
                   listening={listening}
                   api={api}
                 >
+<<<<<<< HEAD
```
Comment (liam-fitzgerald) [inline pkg/interface/src/views/apps/links/app.js:155]: ????

Comment (tylershuster) [reply pkg/interface/src/views/apps/links/app.js:155]: f

**Comment by tylershuster** [comment]
Ok, this should be good. I'm making some changes to spinner because it needs them and suits my purpose.

**Comment by matildepark** [comment]
Still needs parity ... 

before:

image

currently:

image

**Comment by tylershuster** [comment]
Ok, it's just a few pixels off now, but it uses classes instead of explicit pixels.

**Comment by urcades** [comment]
I have a very small design tweak/proposal here, let me know if it's possible/makes sense in the context of this PR:

Screen Shot 2020-09-02 at 4 55 18 PM

As a reference point, the activity being described here can be observed in are.na's block input:

arena

**Comment by tylershuster** [comment]
I like this, but I wonder about it being in this commit. It's already changing a bit of how the S3 component works. I'm happy to turn these designs into reality but I'm getting a little nervous about the current scope.

**Comment by matildepark** [comment]
It would be good to contain this and do the design tweak / additional functions separately yes.

—
~haddef-sigwen
https://urbit.org

**Comment by urcades** [comment]
That's fine! We can ignore this comment and I can situate it in an incremental update. Will check with @matildepark on where best to locate this feedback.

**Comment by tylershuster** [comment]
Would probably pair well with https://github.com/urbit/landscape/issues/74

**Comment by urcades** [comment]
Ah, perfect call. Thanks, will locate this feedback there.

**Review by matildepark — APPROVED** [review]
Clean commits and I'll merge

**Comment by tylershuster** [comment]
These are some messy commits

**Pull request merged**